go/tools/builders/compilepkg.go

```diff
@@ -17,29 +17,17 @@
 package main
 
 import (
-	"bytes"
-	"context"
 	"errors"
 	"flag"
 	"fmt"
 	"io/ioutil"
 	"os"
-	"os/exec"
 	"path"
 	"path/filepath"
 	"sort"
 	"strings"
 )
 
-type nogoResult int
-
-const (
-	nogoNotRun nogoResult = iota
-	nogoError
-	nogoFailed
```

Comment:
Let's keep these constants (maybe put them in a new file constant.go), so we can do `if exitErr.ExitCode() == nogoFailed`, which is more readable.

We probably want to make `nogoSucceeded` to be 0.